### ISSUE TITLE
DPL-401 - As SSRs / Sequencing teams we would like to have improved NovaSeq pipelines inboxes to include the NovaSeq flowcell options as these are currently added in the comments inbox to provide clearer information to the Sequencing team and to reduce the probability of batching errors

### ISSUE THREAD

**Issue by stevieing** [sanger/sequencescape#3617]
**User story**
STORY: Request for new Improved NovaSeq Pipelines Inboxes within Sequencescape
 
This story is regarding the current page in sequencescape:  /pipelines/51
 
Why do we need improved NovaSeq Pipelines Inboxes within Sequencescape?

- Most of faculty sequencing requests are now moving over to NovaSeq (away from HiSeq V4, 4000 and X10)
- There are different NovaSeq flowcell options (SP, S4 and S2) now available to choose from as oppose to just one when the current NovaSeq inbox was designed
- As there is only one inbox available, the only current way to indicate to the sequencing team which flowcell to use, whether the XP is required, and if a custom read length is required is in the COMMENTS
- These comments are easy to miss, and in a busy inbox, the probability of batching errors is now increasing
- In summary having one NovasSeq inbox is now no longer appropriate

 
How could we improve this?

- We (as the SSR team and Sequencing team) would like 3 distinct NovaSeq inboxes to be available – one per flowcell as follows:

  - NovaSeq SP 6000 PE
  - NovaSeq S4 6000 PE
  - NovaSeq S2 6000 PE

- Within these inboxes we would like the following read lengths to be visible if possible:

  - 150bp
  - 100bp
  - OTHER
  - The idea of the OTHER option is to indicate clearly to the sequencing team that the SSR has selected a Custom Read length and therefore they NEED to look at the Comments

- It would also be nice (although not essential) to have a COMMENTS column visible in the inbox too so that the sequencing team can toggle between seeing all the comments and not seeing all the comments maybe (rather than clicking on each one as they have to now)

 
How could we achieve this?

- We would ask for new NovaSeq submission templates to be made as (we believe) the current inboxes are driven by the submission templates
- We would need one NovaSeq submission template for each flowcell type to be available, rather than just NovaSeq 6000 as it is now – so each of the current 6000 submission templates would need to be replaced by:
  - A NovaSeq SP 6000 PE option
  - A NovaSeq S4 6000 PE option
  - A NovaSeq S2 6000 PE option
- Within these templates we would need the accepted read lengths to be:
  - 150bp (as it is now)
  - 100bp (as it is now)
  - OTHER (not yet available)
- We would then still write the custom read length in the COMMENT column in the template

 
**Acceptance Criteria**

- [ ] That the current NovaSeq 6000 PE pipeline is replaced with 3 New Pipelines NovaSeq Sp, NovaSeq S4 and NovaSeq S2 making batching much clearer and easier for the sequencing team and therefore reducing batching errors.
- [ ] That it is very clear (within the inbox) to the sequencing team that a custom read length has been requested.

**Who are the primary contacts for this story**
Siobhan

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by JamesGlover** [comment]
This would require new pipelines and request types as well - all pretty much duplicates of what's there already. It would also impact billing reports, which would need to consider the three request types (although I believe the information from NPG acts as the final arbiter of what the actual flowcell type was).

Might be worth double checking if three distinct inboxes are actually what is needed here, or is we'd be better off adding a new 'metadata' field and improving inbox sorting and batching.

Perhaps predictably, read length is currently an integer column, so wouldn't easily be able to contain 'custom', however I'm not quite sure I understand why this is being suggested, as we could reduce the constraints on the validation and let custom values be provided.

**Comment by JamesGlover** [comment]
Also, we'd need to update the heron submission template to generate the correct request type.

**Comment by JamesGlover** [comment]
Superceded by: https://github.com/sanger/sequencescape/issues/3628
